modules/microshift-config-parameters-table.adoc

```diff
@@ -139,14 +160,49 @@ Not all MIME types benefit from compression, but `HAProxy` uses resources to try
 |`443`
 |Default port shown. Configurable. Valid value is a single, unique port in the `1-65535` range. The values of the `ports.http` and `ports.https` fields cannot be the same.
 
-|`ingress.routeAdmissionPolicy.namespaceOwnership`
+|`ingress.routeAdmissionPolicy`
 |`Strict` or `InterNamespaceAllowed`
-|Describes how hostname claims across namespaces are handled. By default, allows routes to claim different paths of the same hostname across namespaces. Specifying `Strict` prevents routes in different namespaces from claiming the same hostname. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
+|Defines a policy for handling new route claims, such as allowing or denying claims across namespaces. By default, allows routes to claim different paths of the same hostname across namespaces. Specifying `Strict` prevents routes in different namespaces from claiming the same hostname. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
+
+|`ingress.routeAdmissionPolicy.namespaceOwnership`
+|`WildcardsAllowed`, `WildcardsDisallowed`
+|Describes how hostname claims across namespaces should be handled. The default value is `Strict`.
+
+* `Strict`: does not allow routes to claim the same hostname across namespaces.
+* `InterNamespaceAllowed`: allows routes to claim different paths of the same hostname across namespaces.
+
+|`ingress.routeAdmissionPolicy.wildcardPolicy`
+|`WildcardsAllowed`, `WildcardsDisallowed`
+|Describes how routes with wildcard policies are handled by the ingress controller.
+
+* `WildcardsAllowed`: Indicates routes with any wildcard policy are admitted by the ingress controller.
```

Comment:
I see "ingress controller" and "Ingress Controller" versions. Best to stick to one.